docs_src/microservices/application/BuiltIn.md

```diff
@@ -359,7 +362,7 @@ type Filter struct {
 
 !!! example
     ``` go
-    NewFilterFor([] {"Device1", "Device2"}).FilterByDeviceName
+    NewFilterFor([] {"(Device)[0-9]+"}).FilterByDeviceName
```

Comment:
IMO, this line should be added, not replace existing. Also add similar example to other Filter examples.